packages/core/src/artifacts/OwnedAddressTCRWithAppeals.json

```diff
@@ -1183,6 +1183,12 @@
             "events": {},
             "links": {},
             "address": "0x5cf4114912d0b1eacf666e1c6b9fc91eb143956b"
+        },
+        "50": {
```

Comment:
is this local ganache network deployment? do we want to commit this if it's local to your machine?

Reply:
This is local ganache network deployment. It's done in a way that if anyone migrates through `yarn ganache`, it's gonna end up as the same address on the same network id

Reply:
👍